pupa/scrape/base.py

```diff
@@ -5,9 +5,10 @@
 from collections import defaultdict, OrderedDict
 
 import scrapelib
+from django.conf import settings
```

Comment:
Consistency--it's confusing (and hard to patch settings) if some modules import directly and others via django. Seems like there should be a preferred way to import settings, and deviate from it in exceptional cases.